### PR TITLE
Update cn.yaml

### PR DESCRIPTION
**Pull request by TUCAOEVER**
1.Correct some misleading translations.
2.Add missing lines compared to the latest version of fluidd.

Signed-off-by: Yifeng Qiao qiaoyifeng@promc.cn

### CODE REVIEW

**Comment by matmen** [comment]
Hi, thanks for your PR. Before I review this, I don't think you meant to push that workflow config?

**Comment by TUCAOEVER** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by TUCAOEVER** [comment]
> Hi, thanks for your PR. Before I review this, I don't think you meant to push that workflow config?
Oh, sry, my mistake, it should be locale file only.

**Comment by pedrolamas** [comment]
Thank you for your contribution @TUCAOEVER, but before we can merge this, we need the initial comment to have a `Signed-off-by: Your Name <your email address>` footer as per [CONTRIBUTING.md](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md)

**Comment by TUCAOEVER** [comment]
> Thank you for your contribution @TUCAOEVER, but before we can merge this, we need the initial comment to have a `Signed-off-by: Your Name <your email address>` footer as per [CONTRIBUTING.md](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md)

Sure, i have updated my comment.

**Review by pedrolamas — APPROVED** [review]

File: src/locales/cn.yaml [hunk 1]
```diff
@@ -1,39 +1,42 @@
 app:
   bedmesh:
     label:
+      box_scale: 显示比例
       flat_surface: 显示参考平面
       mesh_matrix: 网格矩阵
       probed_matrix: 探测点矩阵
-      profile_name: 配置文件名称
-      remove_profile: 删除 %{name} 配置文件
+      profile_name: 配置文件
```
Comment (matmen) [inline src/locales/cn.yaml:8]: I don't speak chinese, but google translate translates this to "configuration file". Are you sure this translation is correct?

**Review by matmen — APPROVED** [review]
Just one small thing I'm unsure about (through I don't speak chinese so I can't really judge this..), other than that it looks good. Thanks for your contribution!

File: src/locales/cn.yaml (hunk 1 above)
Comment (matmen) [reply src/locales/cn.yaml:8]: Looks like "profile" is an alternate translation to this. LGTM!

**Pull request merged**